src/fides/api/ctl/routes/crud.py

```diff
@@ -33,7 +33,7 @@
 from fides.api.ctl.utils.api_router import APIRouter
 
 # CRUD Endpoints
-routers = []
+routers: List[APIRouter] = []
```

Comment:
I noticed in my testing that the typing wasn't properly recognized so I went ahead and updated this.